lin_actuator.h

```diff
@@ -1,27 +1,37 @@
 /* 
  * File:   LINAC.h
- * Author: zx100
+ * Author: Zachariah Mears
  *
  * Created on February 10, 2019, 12:25 AM
  */
 
 #ifndef LINAC_H
 #define	LINAC_H
 
+#define RED_LED_ON (LATC5 = 0)
```

Comment:
LED macros defined twice. Compiler should complain about this.

Reply:
the compiler isn't complaining, although should I delete the macros in main.c?

Reply:
you should delete the led macros in lin_actuator.h. they don't have anything to do with the valve.

Reply:
by "complain" i mean it should issue a warning along the lines of "macro redefinition at line xx"

Reply:
ok, currently set_DACs uses them

Reply:
I can delete that line though

Reply:
ok you can leave them then